tools/engine_tool/lib/main.dart

```diff
@@ -26,6 +26,13 @@ void main(List<String> args) async {
     return;
   }
 
+  // Fix the pwd so that shelling out to e.g. 'flutter run' works as expected.
+  // TODO(johnmccutchan): Is OLDPWD portable? Probably not.
+  final String? oldPwd = io.Platform.environment['OLDPWD'];
+  if (oldPwd != null) {
+    io.Directory.current = oldPwd;
+  }
```

Comment:
Is this because of the `cd` in the `et` shell entry point? Would a better fix be to remove that `cd` instead?

https://github.com/flutter/engine/blob/afaa6b18ad4e5893086b9fe6146514b6b16a907a/bin/et#L61

If that `cd` is indeed the problem, we should be able to use this instead:

https://github.com/loic-sharma/crank/blob/4b114fed1668a383e1540b2bf4d29dc0b1dc1148/bin/crank#L35-L40

Reply:
+1 Loic

Reply:
Fixed.